build/common/installer/conf/azm-containers-parser-multiline.conf

```diff
@@ -0,0 +1,18 @@
+[MULTILINE_PARSER]
+    name          dotnet-multiline
+    type          regex
+    flush_timeout 1000
```

Comment:
as we discussed add comments for the units of the flush_timeout or use the 1s instead of 1000 .